troubleshoot-install.md

```diff
@@ -50,7 +50,21 @@ If you don’t see a name, you need to add a storage class. For help doing this,
 * AWS: [https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
 * Azure: [https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk](https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk)
 
-Alternatively, you can [deploy Kubecost without persistent storage](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml#L148).
+Alternatively, you can deploy Kubecost without persistent storage to store by following these steps:
+
+> Note: This setup is only for experimental purpose. The metric data is reset when kubecost's pod is rescheduled.
+
+1. On your terminal, run this command to add the kubecost helm repository:
+```helm repo add kubecost https://kubecost.github.io/cost-analyzer/```
+
+1. Next, run this command to deploy Kubecost without persistent storage:
+<pre>
+helm upgrade -i --create-namespace kubecost kubecost/cost-analyzer \
+--namespace kubecost \
+--set persistentVolume.enabled="false" \
+--set prometheus.server.persistentVolume.enabled="false" \
+--set kubecostToken="aGVsbUBrdWJlY29zdC5jb20=xm343yadf98"
```

Comment:
maybe just say <your key> here.

Reply:
Yep, or drop token all together? I think it would reuse on `upgrade`

Reply:
Yeah, or can we just drop the `kubecostToken` line? I believe helm will reuse on `upgrade`

Reply:
Thank you Jesse, Webb. I will drop it in final edit. I wasn't sure if it is required, so I add it to match with what we have in README of Helm chart repository.